cluster/terraform_aks_cluster/config/development.tfvars.json

```diff
@@ -2,12 +2,12 @@
   "cip_tenant": true,
   "kubernetes_version": "1.28.9",
   "default_node_pool": {
-    "node_count": 1,
+    "node_count": 3,
```

Comment:
I think the problem is with the apps not pool, not default. So this one should not be increased